docs/pages/docs/core-concepts/why-turborepo.mdx

```diff
@@ -0,0 +1,19 @@
+# Why Turborepo?
+
+## The problem
+
+Monorepos have many advantages - but **they struggle to scale**. Each workspace you add has its own test suite, its own linting and its own build process. A single monorepo might have **hundreds of tasks to perform**.
```

Comment:
```suggestion
Monorepos have many advantages - but **they struggle to scale**. Each workspace has its own test suite, its own linting and its own build process. A single monorepo might have **hundreds of tasks to execute**.
```

"execute" for consistency "executing tasks" is a recurring phrase everywhere to really drill in by repetition 🏋🏾